app/models/user.rb

```diff
@@ -52,4 +56,16 @@ def to_s
   def user_key
     uid
   end
+
+  def add_curators_as_proxies
+    unless [ User.audit_user_key, User.batch_user_key ].include?(self.user_key)
+      curators = User.group_service.map[CURATOR_GROUP]
+      if curators
+        curators.each do |user_key|
```

Comment:
Consider using a different variable name here b/c `user_key` is a `User` instance variable.